src/common/get.c

```diff
@@ -109,9 +109,6 @@ get_l2protocol(const u_char *pktdata, const uint32_t datalen, const int datalink
         }
 
         if ((pktdata[3] & JUNIPER_FLAG_EXT) == JUNIPER_FLAG_EXT) {
-            if (datalen < 6)
```

Comment:
I removed that validation because there datalen < 4 already https://github.com/appneta/tcpreplay/blob/c785b21891345bf963a9b1c97560573355513b2d/src/common/get.c#L102